packages/docs/src/locales/en/en.json

```diff
@@ -3236,7 +3236,7 @@
   },
   "ag-grid": {
     "title": "AG Grid",
-    "description": "If you need more flexible customization of the display of tables, we recommend using the <a href='https://www.ag-grid.com/vue-data-grid/getting-started'> AG Grid </a> library. Vuestic UI provides a ready-made style theme for this library.",
+    "description": "If you need more flexible customization of the display of tables, we recommend using the [AG Grid](https://www.ag-grid.com/vue-data-grid/getting-started') library. Vuestic UI provides a ready-made style theme for this library.",
```

Comment:
to make link on external source we adds `[[target=_blank]]` to markdown:
`[AG Grid](https://www.ag-grid.com/vue-data-grid/getting-started)[[target=_blank]]`